src/main/cpp/subsystems/Intake.cpp

```diff
@@ -3,3 +3,105 @@
+
+void Intake::Deploy() { m_arm.Set(true); }
+
+void Intake::Stow() { m_arm.Set(false); }
+
+bool Intake::IsDeployed() const { return m_arm.Get(); }
+
+void Intake::SetArmMotor(ArmMotorState armMotorState) {
+    if (armMotorState == ArmMotorState::kIntake) {
+        m_armMotor.Set(1.0);
+    } else if (armMotorState == ArmMotorState::kOuttake) {
+        m_armMotor.Set(-1.0);
+    } else {
+        m_armMotor.Set(0.0);
+    }
+}
+
+void Intake::SetFunnel(double speed) {
+    m_funnelMotorLeft.Set(speed);
+    m_funnelMotorRight.Set(speed);
+}
+
+void Intake::FeedBalls() {
+    m_armMotor.Set(-1.0);
+    SetFunnel(-1.0);
+    m_conveyorMotor.Set(-1.0);
+}
+
+void Intake::SetConveyor(double speed) {
+    m_conveyorMotor.Set(speed); 
+}
+
+bool Intake::IsUpperEnabled() const {
+    return m_upperSensor.GetValue() == kMidProximity;
+}
+
+bool Intake::IsLowerEnabled() const {
+    return m_lowerSensor.GetValue() == kMidProximity;
+}
+
+void Intake::ProcessMessage(const ButtonPacket& message) {
+    if (message.topic == "Robot/AppendageStick2" && message.button == 4 &&
+        message.pressed) {
+        SetArmMotor(ArmMotorState::kIntake);
+    } else if (message.topic == "Robot/AppendageStick2" &&
+               message.button == 6 && message.pressed) {
+        SetArmMotor(ArmMotorState::kOuttake);
+    } else if (message.topic == "Robot/AppendageStick2" &&
+               message.button == 5 && message.pressed) {
+        FeedBalls();
+    } else if (message.topic == "Robot/AppendageStick2" &&
+               message.button == 4 && !message.pressed) {
+        SetArmMotor(ArmMotorState::kIdle);
+    } else if (message.topic == "Robot/AppendageStick2" &&
+               message.button == 6 && !message.pressed) {
+        SetArmMotor(ArmMotorState::kIdle);
+    } else if (message.topic == "Robot/AppendageStick2" &&
+               message.button == 5 && !message.pressed) {
+        SetArmMotor(ArmMotorState::kIdle);
+        SetFunnel(0.0);
+        m_conveyorMotor.Set(0.0);
+    }
+    if (message.topic == "Robot/AppendageStick2" && message.button == 3 &&
+        message.pressed) {
+        if (IsDeployed()) {
+            Stow();
+        } else {
+            Deploy();
+        }
+    }
+}
+
+void Intake::SubsystemPeriodic() {
+    switch (m_state) {
+        case State::kIdle: {
+            if (m_armMotor.Get() > 0.0) { // Checks to see if the intake is running
+                SetFunnel(1.0);
+                SetConveyor(-1.0);
+                m_state = State::kDropBalls;
+            }
+            break;
+        }
+        case State::kDropBalls: {
+            if (IsLowerEnabled()) { // Checks to see if the proximity sensor detects a ball
+                SetConveyor(1.0);
+                m_state = State::kRaiseBalls;
+            }
+            break;
+        }
+        case State::kRaiseBalls: {
+            if (IsUpperEnabled()) { // Checks to see if the proximity sensor detects a ball
```

Comment:
Put the comments on the line before if statements instead of at the end of the same line. It's easier to read and gives you more space to write the comments.